### DISCUSSION TITLE
2009/05/07/%E9%87%8D%E6%96%B0%E6%88%AA%E5%8F%96%E4%B9%A6%E4%B8%AD%E6%8F%92%E5%9B%BE

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#258] [Announcements]
# 2009/05/07/%E9%87%8D%E6%96%B0%E6%88%AA%E5%8F%96%E4%B9%A6%E4%B8%AD%E6%8F%92%E5%9B%BE

本以为交稿就万事大吉了，结果才没轻松几天，就又要忙了。收到编辑的信说书稿中图片分辨率太低，需要重新截取。虽然打开word文档看那些插图还蛮清楚的，但放在排版软件中就非常模糊了。估计是因为常用的排版软件和word文档的兼容性不好，在格式转换时，图片质量损失严重。我现在只好再把所有插图重新存为TIFF-无压缩RGB格式的文件。

https://blog.qizhen.xyz/2009/05/07/%E9%87%8D%E6%96%B0%E6%88%AA%E5%8F%96%E4%B9%A6%E4%B8%AD%E6%8F%92%E5%9B%BE

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Yigang
2009年05月8日 09:10 编辑
Ruan兄可以试试FastStone Capture，很好用的抓图软件

回复
